configure.ac

```diff
@@ -345,13 +345,12 @@ if test "$enable_logind" = "yes"; then
 fi
 AM_CONDITIONAL(ENABLE_LOGIND, test "x$enable_logind" != "xno")
 
-AC_SUBST(LIBCRYPT)
-AC_CHECK_LIB(crypt, crypt, [LIBCRYPT=-lcrypt],
+# FIXME check whether the next variable really used somewhere 
+AC_CHECK_LIB(crypt, crypt,[LIBCRYPT=-lcrypt
```

Comment:
Why remove the space?

Reply:
That was introduced in https://github.com/shadow-maint/shadow/commit/5cd04d03f94622c12220d4a6352824af081b8531#diff-49473dca262eeab3b4a43002adb08b4db31020d190caaad1594b47f1d5daa810R413

I have a feeling the intention was to check whether yescrypt is supported by libcrypt.so. But Following snippet does not do that.
```
AC_SUBST(LIYESCRYPT)
AC_CHECK_LIB(crypt, crypt, [LIYESCRYPT=-lcrypt],
	[AC_MSG_ERROR([crypt() not found])])

```

Reply:
Anyway, this particular macros is full duplication of another check + addition of unused substitution. 
Should be removed.